app/views/assessments/_edit_problems.html.erb

```diff
@@ -1,6 +1,15 @@
-<h2>Edit Problems for <%= current_assessment_link %></h2>
+<h3>Edit Problems for <%= current_assessment_link %></h3>
+<br>
 <ul>
-  <% # this bit of Rails magic renders app/views/problems/_problem.html.erb for every problem in the collection %>
-  <%= render @assessment.problems %>
```

Comment:
I wanted to edit the HTML, but not sure if 'app/views/problems/_problem.html.erb' is used somewhere else.